google-cloud-bigquery/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://mojo.codehaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <className>com/google/cloud/bigquery/QueryParameterValue</className>
+    <method>java.util.Map getStructTypes()</method>
```

Comment:
Is space allowed inside `<method>` tag?

Reply:
yes if you are getting an error than that it will shows in terminal, so whatever errors you get from that just copy that and paste into method tag, that's it.